src/cxoCursor.c

```diff
@@ -1604,11 +1601,12 @@ static PyObject *cxoCursor_multiFetch(cxoCursor *cursor, int rowLimit)
 static PyObject *cxoCursor_fetchOne(cxoCursor *cursor, PyObject *args)
 {
     uint32_t bufferRowIndex = 0;
-    int found = 0;
```

Comment:
I'd prefer to keep this (and similar) initializations; not all static analyzers are smart.

Reply:
Ignore this!